iron-flex-layout.html

```diff
@@ -58,9 +58,21 @@
 @pseudoElement iron-flex-layout
 @demo demo/index.html
 -->
+<script>
+  // This is left only for backward compatibility with projects
+  // that incorrectly relied on unscoped global [hidden] rules;
+  // removing would be a breaking change, but new projects
+  // should never rely on this.
+  (function() {
+    if (Polymer.Settings && !Polymer.Settings.useNativeShadow) {
```

Comment:
Also, `useNativeShadow` looks to be only a 1.x thing. 

Is making this conditional accomplishing much good?  The least breaking change is to leave it unconditional like it was.